lib/domain/service/call.dart

```diff
@@ -161,6 +161,9 @@ class CallService extends DisposableService {
       Rx<OngoingCall>? call;
 
       try {
+        if (chat?.unreadCount.value == 1) {
+          await _chatService.readAll([chatId]);
+        }
```

Comment:
Will this be sufficient enough? Can't there be a race condition? For example, If we accept a call fast enough and this single unread message was a real `ChatMessage`? Or am I being too deep and this condition isn't realistic enough to do something about it?